crates/bitwarden-core/src/client/encryption_settings.rs

```diff
@@ -30,10 +30,20 @@ pub enum EncryptionSettingsError {
     MissingPrivateKey,
 }
 
+/// A struct containing the core keys of a Bitwarden user.
 #[derive(Clone)]
 pub struct EncryptionSettings {
+    /// The users symmetric key, stored on the server after being
+    /// encrypted with another key.
```

Comment:
Is these details relevant to the EncryptionSettings context? We could expand this to give the bigger picture in which case this information could be relevant. If not I think we should focus on the EncryptionSettings perspective which is not to share this anywhere.

```suggestion
    /// The user's primary crypto key.
    ///
    /// Every Bitwarden user has a symmetric user_key which acts as the encryption key to unlock everything else.
    /// .......
    /// This key is generally protected by a [MasterKey] derived from the users password, or .... and uploaded to the server.
    ///
    /// ## References:
    /// - https://bitwarden.com/help/account-encryption-key/
    /// <confluence link>?
```